commands/cloudapi-v6/backupunit.go

```diff
@@ -427,12 +427,13 @@ func RunBackupUnitDelete(c *core.CommandConfig) error {
 	}
 
 	backupunitId := viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgBackupUnitId))
+	backupunitName := viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgName))
```

Comment:
Again, this flag doesn't exist for `backupunit delete`, you need to `findById` to retrieve the name